docs/integrations/github/discovery.md

```diff
@@ -173,6 +173,8 @@ If you do so, `default` will be used as provider ID.
 - **`organization`**:
   Name of your organization account/workspace.
   If you want to add multiple organizations, you need to add one provider config each.
+- **`namespace`** _(optional)_:
+  The namespace to assign to ingested entities. Uses the `default` namespace if not provided.
```

Comment:
Best to highlight that this only applies to the location entities right? Not the actual entities themselves

Reply:
@Rugvip ah, that's sort of my question;  My goal here is to make sure the _generated_ entities (`Components` etc) are added to this namespace.  Is this the right spot for modification?  as I mentioned in the PR, I couldn't find the place in the processing/provdier loop where locations were fetched.

Reply:
@Rugvip after some digging this weekend, I think that #25814 handles this problem more gracefully.  I'm closing this one in favor of that.  Thanks for your feedback thus far!